### PR TITLE
Boost launch modal + misc fixes [CIVIL-862]

### PR DESCRIPTION
**Pull request by tobek**
This is needed for forthcoming embed code stuff.

Lots of storyshots updates cause I updated `<Button>`.

### CODE REVIEW

File: packages/components/src/icons/ShareTwitterIcon.tsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ export const ShareTwitterIcon = () => {
     <svg height="32" viewBox="0 0 32 32" width="32" xmlns="http://www.w3.org/2000/svg">
       <path
         d="m28.954 8.569c-.885.389-1.83.654-2.825.775 1.014-.611 1.794-1.574 2.163-2.723-.951.555-2.005.959-3.127 1.184-.896-.959-2.173-1.559-3.591-1.559-2.717 0-4.92 2.203-4.92 4.917 0 .39.045.765.127 1.124-4.09-.193-7.715-2.157-10.141-5.126-.427.722-.666 1.561-.666 2.475 0 1.71.87 3.213 2.188 4.096-.807-.026-1.566-.248-2.228-.616v.061c0 2.385 1.693 4.374 3.946 4.827-.413.111-.849.171-1.296.171-.314 0-.615-.03-.916-.086.631 1.953 2.445 3.377 4.604 3.417-1.68 1.319-3.809 2.105-6.102 2.105-.39 0-.779-.023-1.17-.067 2.189 1.394 4.768 2.209 7.557 2.209 9.054 0 13.999-7.496 13.999-13.986 0-.209 0-.42-.015-.63.961-.689 1.8-1.56 2.46-2.548z"
-        fill="#8b8581"
-        className="twitter-share-icon"
+        fill="#191919"
+        fill-rule="nonzero"
```
Comment (sruddy) [inline packages/components/src/icons/ShareTwitterIcon.tsx]: `fillRule`

File: packages/components/src/icons/ShareEmailIcon.tsx [hunk 2]
```diff
@@ -3,14 +3,23 @@ import * as React from "react";
 export const ShareEmailIcon = () => {
   return (
     <svg height="32" viewBox="0 0 32 32" width="32" xmlns="http://www.w3.org/2000/svg">
```
Comment (sruddy) [inline packages/components/src/icons/ShareEmailIcon.tsx]: Same here:
```<svg height="32" viewBox="0 0 32 32" width="32" xmlns="http://www.w3.org/2000/svg">
  <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
    <polygon points="4 4 28 4 28 28 4 28"></polygon>
    <path
      d="M24,8 L8,8 C6.9,8 6.01,8.9 6.01,10 L6,22 C6,23.1 6.9,24 8,24 L24,24 C25.1,24 26,23.1 26,22 L26,10 C26,8.9 25.1,8 24,8 Z M23.6,12.25 L16.53,16.67 C16.21,16.87 15.79,16.87 15.47,16.67 L8.4,12.25 C8.15,12.09 8,11.82 8,11.53 C8,10.86 8.73,10.46 9.3,10.81 L16,15 L22.7,10.81 C23.27,10.46 24,10.86 24,11.53 C24,11.82 23.85,12.09 23.6,12.25 Z"
      fill="#191919"
      fillRule="nonzero"
    ></path>
  </g>
</svg>```

File: packages/components/src/icons/EmbedIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+
+export const EmbedIcon = () => {
+  return (
+    <svg width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg">
```
Comment (sruddy) [inline packages/components/src/icons/EmbedIcon.tsx]: All those transforms counteract each other, you can optimize that:
```
<svg width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg">
  <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
    <polygon points="4 4 28 4 28 28 4 28"></polygon>
    <path
      d="M9.42130253,10.2684345 C10.3266052,9.36313187 11.7034439,10.7399706 10.7981413,11.6452732 L6.35806996,16.0849552 L10.7981413,20.5250265 C11.7034439,21.4303291 10.3266052,22.8071678 9.42130253,21.9018652 L4.29281186,16.7733745 C3.91277945,16.3933421 3.91277945,15.7765682 4.29281186,15.3965358 L9.42130253,10.2684345 Z M17.0772424,7.74721537 C17.4074345,6.51211003 19.2857914,7.01401759 18.9555993,8.2495123 L14.6222174,24.4234737 C14.2920253,25.658579 12.4136684,25.1566715 12.7438605,23.9211768 L17.0772424,7.74721537 Z M20.9017079,11.6452732 C19.9964053,10.7399706 21.373244,9.36313187 22.2785466,10.2684345 L27.4070373,15.3965358 C27.7870697,15.7765682 27.7870697,16.3933421 27.4070373,16.7733745 L22.2785466,21.9018652 C21.373244,22.8071678 19.9964053,21.4303291 20.9017079,20.5250265 L25.3417792,16.0849552 L20.9017079,11.6452732 Z"
      fill="#191919"
      fillRule="nonzero"
    ></path>
  </g>
</svg>
```

Comment (tobek) [reply packages/components/src/icons/EmbedIcon.tsx]: Good catches thanks, was just copy-pasting svg assets. Strange that `fill-rule` wasn't caught though I seem to remember some part of linting catching malformed names.

**Comment by dankins** [comment]
:tada: This PR is included in version 4.8.20 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**